multimodal/tests/unittests/others/test_object_detection.py

```diff
@@ -148,6 +149,16 @@ def test_mmdet_object_detection_inference_df(checkpoint_name):
     data_dir = download_sample_dataset()
 
     test_path = os.path.join(data_dir, "Annotations", "test_cocoformat.json")
+
+    test_path_without_annotations = os.path.join(data_dir, "Annotations", "test_cocoformat_nolabel.json")
+
+    with open(test_path, "r") as f:
+        test_data = json.load(f)
+    test_data.pop("annotations")
+    test_data["images"] = test_data["images"][:100]
+    with open(test_path_without_annotations, "w+") as f_wo_ann:
+        json.dump(test_data, f_wo_ann)
+
```

Comment:
Can we modify the predictions in tutorials to reflect this support?

Reply:
Updated the doc a bit. I'll add a new tutorial on what kind of data we can take for prediction (inference).